testing/android/run_tests.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+set -e
+
+pushd android_host_app
+
+# Required to create gradlew script
+if [ "$FRAMEWORK_PATH" = "" ]
+then
+  FLUTTER_BIN=flutter
+else
+  FLUTTER_BIN=$FRAMEWORK_PATH/flutter/bin/flutter
+fi
+# Creates android/gradlew
+$FLUTTER_BIN build apk --debug
```

Comment:
TODO(@matthew-carroll): split this into 2 projects with Android and Flutter. Then we can use include_flutter.groovy and test that path. Then we won't need to build here first.

Reply:
TODO(@matthew-carroll): we need to make include_flutter.groovy pass in the local-engine here. Seems like flutter.gradle already somehow accounts for it via the gradle project property 'localEngineOut'. 

For now, we can just set the --local-engine CLI arg like the other stuff in .cirrus.yml does